### DISCUSSION TITLE
[vue-tsc] Support for slot argument types?

### DISCUSSION THREAD

**Discussion by darthf1** [vuejs/language-tools#2017] [Q&A]
Hi!

Vue: 2.7.13
vue-tsc: 1.0.7 (`vue-tsc --noEmit --strict`)

I'm using a third party Vue component[ written in javascript](https://github.com/vuelidate/vuelidate/blob/next/packages/components/src/ValidateEach.js), and I am trying to create a typescript definition. Issue link: https://github.com/vuelidate/vuelidate/issues/900#issuecomment-1277369291.

I am trying to create a definition, which also types the arguments in the `default slot` and I'm having zero result so far. I also tried the example given on https://logaretm.com/blog/generic-type-components-with-composition-api/, but whatever I do the errors reported by `vue-tsc` are the same.

![image](https://user-images.githubusercontent.com/17253332/195806362-f24fb4b2-ea4b-45d6-85ff-2792ccb8d514.png)

![image](https://user-images.githubusercontent.com/17253332/195806319-f82cb8e1-7c4c-469b-9184-d63072353406.png)

This leads me to the question: does `vue-tsc` support typings of slot arguments?

**Comment by johnsoncodehk** [comment]
You can reference to https://twitter.com/johnsoncodehk/status/1580608720069087232?s=20&t=mZFHYaVG40dynbtmxN1gCg
